lib/modules/dosomething/dosomething_static_content/dosomething_static_content.module

```diff
@@ -50,7 +50,8 @@ function dosomething_static_content_preprocess_node(&$vars) {
             break;
 
           case 'text_formatted':
-            $vars[$label] = $field->value->value();
+            global $user;
+            $vars[$label] = dosomething_helpers_extract_field_data($vars['node']->{$field_name}, $user->language);
```

Comment:
i think this language thing might cause some wonkyness since we didn't translate static content.
there might not be content in the user's language